pycalphad/core/equilibrium.py

```diff
@@ -267,7 +267,9 @@ def equilibrium(dbf, comps, phases, conditions, output=None, model=None,
                 undefs = list(out.atoms(Symbol) - out.atoms(v.StateVariable) - set(param_symbols))
                 for undef in undefs:
                     out = out.xreplace({undef: float(0)})
-                cf, gf, hf = build_functions(out, tuple([v.P, v.T] + site_fracs), parameters=param_symbols)
+                cf, gf = build_functions(out, tuple([v.P, v.T] + site_fracs),
```

Comment:
@richardotis can you clarify: we aren't building Hessians, but handling for Hessian code are still around. Can you be more specific on your comments in the main PR comment?

Reply:
I hesitated to remove the Hessian handling code altogether, but since history lives on in Git, I should push forward and clean it up.